driver/razerkbd_driver.c

```diff
@@ -1708,12 +1821,6 @@ static int razer_raw_event(struct hid_device *hdev, struct hid_report *report, u
         return 0;
     }
 
-    // ...and on these keyboards
-    switch (usb_dev->descriptor.idProduct) {
-    case USB_DEVICE_ID_RAZER_HUNTSMAN_ELITE:
-        return 0;
-    }
-
```

Comment:
Why did you remove this?

Reply:
Yes, this change has been made for support Multimedia buttons in Huntsman Elite (they are the same as Blackwidow Elite)